public/pages/workflow_detail/workflow_detail.test.tsx

```diff
@@ -178,3 +169,67 @@ describe('WorkflowDetail Page Functionality (Custom Workflow)', () => {
     expect(history.location.pathname).toBe('/workflows');
   });
 });
+
+describe('WorkflowDetail Page with skip ingestion option (Hybrid Search Workflow)', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  test(`renders the WorkflowDetail page with skip ingestion option`, async () => {
+    const {
+      container,
+      getByTestId,
+      getAllByText,
+      getAllByTestId,
+    } = renderWithRouter(workflowId, workflowName, WORKFLOW_TYPE.HYBRID_SEARCH);
+
+    // "Create an ingest pipeline" option should be selected by default
+    const createIngestRadio = container.querySelector('#create');
+    expect(createIngestRadio).toBeChecked();
+
+    // "Skip ingestion pipeline" option should be unselected by default
+    const skipIngestRadio = container.querySelector('#skip');
+    expect(skipIngestRadio).not.toBeChecked();
+
+    // Selected "Skip ingestion pipeline"
+    await waitFor(() => userEvent.click(skipIngestRadio!));
```

Comment:
What is the reasoning for the `waitFor`s on the clicks now? I thought it was determined to have them on the followup assertions?

Reply:
@ohltyler, Placed **waitFor** in all tests for the first followup assertions of click events.